k2/addons/k2-upgrade/upgrade/__init__.py

```diff
@@ -0,0 +1,24 @@
+from zaf.config.options import ConfigOptionId
+from zaf.config.types import ConfigChoice
+from zaf.messages.message import MessageId
+
+from k2.sut import SUT
+
+K2_UPGRADE_COMPONENT = 'Upgrader'
```

Comment:
Not used.

Reply:
It's used by my unpublished "mender" addon :innocent: I thought the name should be introduced together with the interface, even before any addons actually implements/extends it